cpp/modmesh/universe/World.hpp

```diff
@@ -57,18 +57,24 @@ class World
     using real_type = T;
     using value_type = T;
     using size_type = typename NumberBase<int32_t, T>::size_type;
-    using vector_type = Point3d<T>;
-    using vertex_type = vector_type;
+    using point_type = Point3d<T>;
```

Comment:
Rename `vertex_type` to `point_type` and remove the incorrect type name `vector_type`.